api/backend/ecs/utilities.go

```diff
@@ -187,10 +188,6 @@ var GetLogs = func(cloudWatchLogs cloudwatchlogs.Provider, taskARNs []*string, t
 
```

Comment:
also, you have debug statements left over in the code you forgot to remove last time